mypy.ini

```diff
@@ -17,4 +22,41 @@ check_untyped_defs       = False
 disallow_incomplete_defs = False
 no_implicit_optional     = False
 warn_return_any          = False
-warn_unreachable         = False
+
+# Disable warn_unreachable for some modules until we clean them up
```

Comment:
I'll follow this up with separate PRs for the remaining cases, as they require more input and I'm trying to keep this PR to the low-hanging fruit.